tox.ini

```diff
@@ -47,3 +48,10 @@ commands =
 [pytest]
 filterwarnings =
     ignore:.*async with lock.* instead:DeprecationWarning
+
+[testenv:security]
+deps =
+    bandit
+
+commands =
+    bandit --recursive sanic --skip B404,B101 --exclude sanic/reloader_helpers.py
```

Comment:
What is the thought behind skipping this file?

Reply:
So that file uses subprocess, which will make the build fail. 

I can add multiple #nosec comments, but this caused issues with black formatter.

I don't want to blanket exclude the specific error in case it pops up anywhere else.

Any suggestions @ahopkins

Reply:
Nope. Let's leave it out for now. That file is in some serious need of attention anyway. No need to tackle it here.

Reply:
Great. I would be willing to have a look at that file if you have any pointers of where to begin. I've had some idea, but don't want to run down a rabbit hole.